### PR TITLE
Fix up of bit library

### PR DESCRIPTION
**Pull request by jdpatdiscord**

### CODE REVIEW

**Comment by ShowerHead-FluxTeam** [comment]
I ShowerHeadFD, approve this request.

**Review by LuaOranges — CHANGES_REQUESTED** [review]

File: api/bit/bit.arshift.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# bit.band
```
[the remark this answers is not in the document]
Comment (jdpatdiscord) [reply api/bit/bit.arshift.md]: This file is now fixed up.

File: RuntimeCompliance.lua [hunk 2]
```diff
@@ -0,0 +1,13 @@
+for _, o in next, {"bit", "cache"} do
```
[the remark this answers is not in the document]
Comment (jdpatdiscord) [reply RuntimeCompliance.lua]: Removed the file. Will add it in the future

**Comment by LuaOranges** [comment]
Looks good. As this is mostly just a cleanup PR I'll merge it now.

**Pull request merged**